### DISCUSSION TITLE
Validation in JSON Repeater

### DISCUSSION THREAD

**Discussion by Fusseldieb** [directus/directus#2979] [Feature Requests]
Hi,

I've seen that in almost all fields we have a RegEx validation, but in the Repeater this option lacks. 

Would be nice to see that in v9 😊

Greetings!

**Comment by rijkvanzanten** [comment]
Do you mean field validation for the individual fields of each row?

**Reply by Fusseldieb** [reply]
Yep. 

For example, the Tags input has a RegEx Validation for every tag and the Text input has a RegEx Validation for the whole field, but the repeater has none.

**Reply by rijkvanzanten** [reply]
You're talking about v8 in this case right?

**Reply by Fusseldieb** [reply]
Neither v8 nor v9 has this

![image](https://user-images.githubusercontent.com/4715129/98832004-103bbf00-241b-11eb-8283-43d56940300f.png)

**Reply by rijkvanzanten** [reply]
I meant your previous statement 😉 

> For example, the Tags input has a RegEx Validation for every tag and the Text input has a RegEx Validation for the whole field, but the repeater has none.

**Reply by Fusseldieb** [reply]
> I meant your previous statement 😉

Yes, I'm talking about the Repeater, about the individual fields ahaha

**Comment by Michael-Nussbaumer** [comment]
I was looking for RegEx Validation but couldn't find how to achive it. I cannot find the Validation Field in v9.

And the validation via Permissions isn't working for me, also it would be the wrong place to have validation of the field value on Role Permission Level I guess.

**Reply by benhaynes** [reply]
ref: https://github.com/directus/next/issues/969

**Reply by Michael-Nussbaumer** [reply]
updated ref: https://github.com/directus/directus/issues/2916

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
